WcaOnRails/app/assets/javascripts/char-counter.js

```diff
@@ -0,0 +1,13 @@
+$(() => {
```

Comment:
We don't have babel set up for this part of assets (it's kinda legacy), could you switch that to a `function`?
Also please update tabs to 2 spaces for the sake of consistency =)

Reply:
Done.